specification/_global/search/_types/hits.ts

```diff
@@ -39,7 +39,7 @@ import { Sort, SortResults } from '@_types/sort'
 
 export class Hit<TDocument> {
   _index: IndexName
-  _id: Id
+  _id?: Id
```

Comment:
This one is super-specific, the only time from what I can see where `_id` is optional on a Hit is when `stored_fields: _none_` which explicitly asks for zero fields on hits.